releasenotes.txt

```diff
@@ -1,4 +1,27 @@
-﻿Build 5.4.1
+﻿Build 5.4.2
+=============================
+
+Release notes - NHibernate - Version 5.4.2
+
+6 issues were resolved in this release.
```

Comment:
nit: probably need to mention that it includes fixes from 5.3.16 as well.

Reply:
I think easiest way would be to create a task 'Merge 5.3.16 into 5.4.x' and mark it as closed by merge commit.